gx_suite_definitions/gene_info.ipynb

```diff
@@ -343,6 +343,20 @@
     "validator.expect_column_values_to_match_json_schema(\"ensembl_info\", ensembl_info_schema)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# uniprotkb_accessions\n",
+    "validator.expect_column_values_to_be_of_type(\"uniprotkb_accessions\", \"list\")\n",
+    "validator.expect_column_values_to_have_list_members_of_type(column=\"uniprotkb_accessions\", member_type=\"str\")\n",
+    "\n",
+    "# Regex from https://www.uniprot.org/help/accession_numbers\n",
+    "# validator.expect_column_values_to_match_regex(\"uniprotkb_accession\", \"[OPQ][0-9][A-Z0-9]{3}[0-9]|[A-NR-Z][0-9]([A-Z][A-Z0-9]{2}[0-9]){1,2}\")"
```

Comment:
Is this commented out for a reason? I feel like I remember us having some issues with the regex for UniProt accessions in a different PR but I can't remember if we figured it or gave up...

Reply:
I commented it out because I was not able to figure out how to perform a regex match for each value of a list using gx validation. I'll remove it.

Reply:
If you wanted to do this you would need to use the JSON schema validation expectation https://github.com/Sage-Bionetworks/agora-data-tools/pull/160#pullrequestreview-2458858218